docs/style/js/catalog/common.js

```diff
@@ -25,7 +25,7 @@ var brooklyn = (function ($, _) {
         },
 
         entityCard: _.template(
-            "<a class='plain' data-type='<%= type %>' href='entities/<%= type %>.html'>" +
+            "<a class='plain' data-type='<%= type %>' href='brooklyn-node.html#entities/<%= type %>'>" +
```

Comment:
Not an expert in SEO, but wouldn't this hamper findability? Technically it makes more sense, but having a separate page for each entity might be more appropriate google-wise.